operator/pkg/resources/secret.go

```diff
@@ -95,11 +95,12 @@ func (r *PreStartStopScriptResource) update(
 	logger logr.Logger,
 ) error {
 	// Do not touch existing last-applied-configuration (it's not reconciled in the main loop)
-	if val, ok := current.Annotations[LastAppliedConfigurationAnnotationKey]; ok {
+	// TODO: what's this doing on here?
```

Comment:
This check can be removed. Secrets are not annotated.

Reply:
Done